core/modules/parsers/audioparser.js

```diff
@@ -17,7 +17,8 @@ var AudioParser = function(type,text,options) {
 			type: "element",
 			tag: "audio",
 			attributes: {
-				controls: {type: "string", value: "controls"}
+				controls: {type: "string", value: "controls"},
+				style: {type: "string", value: "width: 100%; object-fit: contain"}
```

Comment:
Hi @donmor apologies for not noticing before, but it's actually possible to pass the styles directly:

```
tag: "audio",
attributes: {
	controls: {type: "string", value: "controls"}
},
style: {
	width: "100%",
	"object-fit": "contain"
}
```

Reply:
So how to pass the styles to the embedded video or audio? For example, I imported a video file, and how to insert it into a tiddler and make it fit the width of the page?

Reply:
Hi @donmor apologies for being unclear. I'm pointing out an alternative way of passing the style properties to the domMaker. As you can see from the source, it supports a "style" property on the options object for passing a hashmap of styles:

https://github.com/Jermolene/TiddlyWiki5/blob/master/boot/boot.js#L159-L184

Reply:
But the only way I know to insert a video tiddler to another tiddler is using transclusion like:
`{{video.mp4}}`
and the video inserted keeps its original size.
And then, how can I make the video fit the tiddler?

Reply:
Hi @donmor the change I am suggesting is just a different way of assigning the same style properties to the element.

Reply:
You mean...put the style things to another position?

Reply:
Yes, it's just a different way of passing things to domMaker.

Reply:
I just tried your code but not working. The video inserted still keeps its original size.

Reply:
Hi @donmor apologies, that's entirely my mistake, the syntax I'd suggested doesn't work here.

Reply:
You mean I should keep the previous code?

Reply:
To be honest I think the previous code "as plain as texts":)